core/dbt/task/run_operation.py

```diff
@@ -108,5 +110,6 @@ def run(self) -> RunResultsArtifact:
 
         return results
 
-    def interpret_results(self, results):
+    @classmethod
```

Comment:
Why this becomes a class method where all other places are still just a normal method?

Reply:
Cool! I think making things more consistent is not in scope of this PR so current implementation works!